screens/TimerScreen.js

```diff
@@ -15,7 +15,9 @@ const TimerScreen = ({ navigation }) => {
   const [sound, setSound] = useState();
   const [round, setRound] = useState(0);
   const [start, setStart] = useState(false);
-
+  const [isVisible, setVisible] = useState();
+  const [time, setTime] = useState(0);
+  const [mode, setMode] = useState(0);
```

Comment:
mode should be a string I think then you can have maxRounds to go with it, or just have mode called maxRounds if you wanna add it the ability to have times like 75 and stuff

Reply:
you could have it just be a dropdown or something before starting